### ISSUE TITLE
[Feature Request] Improve accessibility for table sorting

### ISSUE THREAD

**Issue by ijdickinson** [ElemeFE/element#6869]
### Existing Component
Yes

### Component Name
el-table-column

### Description
The sort affordance on `el-table` columns presents a caret, consisting of two small triangles, to both show the current state of the sort (ascending, descending, not sorted) *and* provide a control for changing the sort order on the column. In order for a user to change the sort order, they have to click on a small click target that is approximately 15px square using the default theme.

I think this is a problem, because:

* it's hard enough for ordinary users, with a mouse, to hit such a small target accurately
* on a touch-screen device, it's very easy for a user to accidentally touch the adjacent control, and not get the sort effect they want
* for users with accessibility requirements, e.g reduced vision or restricted motor control, seeing and accurately hitting a very small target is particularly difficult. It would likely make the interface fail accessibility audit.

In [jQuery datatables](https://datatables.net/), a similar caret design is used to *display* the current sort order. However, to *select* the current sort order, the user clicks or taps anywhere on the column heading. These events cycle through the various sort orders in sequence (ascending, descending, none, ascending, etc). Since the column heading is larger than the caret, it's much easier to hit. This design does have problems with discoverability (users don't always realise they can click anywhere on the heading), so another possible alternative is to have a pop-up dialog for sort preference.

**Comment by Leopoldthecoder** [comment]
> the user clicks or taps anywhere on the column heading

This also applies to Element's Table

> These events cycle through the various sort orders in sequence (ascending, descending, none, ascending, etc)

Element's Table behaves the same way.

![untitled](https://user-images.githubusercontent.com/10095631/30059998-0ee1e468-9274-11e7-9645-e493dc17eac4.gif)

**Comment by ijdickinson** [comment]
So they do. I did try that earlier, and it wasn't working as I expected, but now it is. User error! Apologies :)

**Comment by wacky6** [comment]
@Leopoldthecoder 

Maybe add 'cursor: pointer' to column header's text and arrows (or the entire cell) ?

**Comment by Leopoldthecoder** [comment]
@wacky6 That we can do.

**Comment by Leopoldthecoder** [comment]
#6874
